src/components/layout/playground/applyForm.tsx

```diff
@@ -319,6 +319,16 @@ const MainForm: React.FC<RequestProps> = ({ request }) => {
   useOnce(
     () => {
       if (!lastApplication) return;
+
+      if (lastApplication?.status === 'Blocked') {
+        return toast.error(
+          'Please contact us before applying for more requests.',
+          {
+            onClose: () =>
+              router.push('/contact', undefined, { shallow: true }),
+          }
```

Comment:
A toast can easily be missed, to allow the user to navigate on their own, please show this message where the form is currently placed and put a contact button below it. Like this 
![grafik](https://user-images.githubusercontent.com/45602265/216173070-2d8ac404-0cdb-4e09-9fba-89feaa7e23c4.png)

example code
```jsx
  if (isBlocked) {
    return (
      <>
        <h1 className="text-3xl">
          ⚠️ Please contact us before applying for more requests.
        </h1>
        <DarkButton className="w-fit m-auto mt-5" href="/contact">
          Contact
        </DarkButton>
      </>
    );
  }
```